### DISCUSSION TITLE
WebcamInfo redundant data

### DISCUSSION THREAD

**Discussion by BlueRyse** [noi-techpark/it.bz.opendatahub.databrowser#196] [Q&A]
@sseppi @gappc @pkritzinger 

Hello everyone,
Today I discovered that in the webcamInfo detail view there are two places where Gps Data is shown. As you can see in the picture, the Latitude, Longitude, and Altitude data is shown both in the GPS Data category and in the webcam details one.

I would suggest to show this Data only once, but let me know if you want to keep it this way.
![Screenshot from 2022-11-03 15-10-02](https://user-images.githubusercontent.com/57491067/199743063-daa64d43-53dc-4744-944b-ecdf480f8384.png)
![Screenshot from 2022-11-03 15-07-21](https://user-images.githubusercontent.com/57491067/199742728-9a4dcba0-9855-4926-8b9a-bccabb8e4555.png)

**Comment by sseppi** [comment]
@BlueRyse  Thank you for sharing this finding.

@RudiThoeni do you know if the two GPS coordinates are always the same? Or could it be that like the beacon we have the GPS coodinates where the webcam is installed and the one of the POI?

**Comment by BlueRyse** [comment]
@sseppi Let me answer this, if I may. This Data is taken from the same attribute in the API (GPSPoints), so both are always going to be the same in every record.

**Comment by pkritzinger** [comment]
@BlueRyse @sseppi If data remains always the same, I would only show the information once by removing it from the section "Webcam Details". Hence, all GPS-related attributes are shown in the section "GPS Data".
